services/credential-schema/src/schema/schema.controller.ts

```diff
@@ -0,0 +1,138 @@
+import {
+  Body,
+  CacheInterceptor,
+  CACHE_MANAGER,
+  Controller,
+  Get,
+  Inject,
+  Param,
+  Patch,
+  Post,
+  Query,
+  UseInterceptors,
+  Put,
+} from '@nestjs/common';
+import {
+  ApiBadRequestResponse,
+  ApiBody,
+  ApiCreatedResponse,
+  ApiNotFoundResponse,
+  ApiOkResponse,
+  ApiOperation,
+  ApiParam,
+  ApiQuery,
+} from '@nestjs/swagger';
+import { Cache } from 'cache-manager';
+
+import { CreateCredentialDTO } from './dto/create-credentials.dto';
+import { VCItem } from './entities/VCItem.entity';
+import { VCModelSchema } from './entities/VCModelSchema.entity';
+import { SchemaService } from './schema.service';
+
+@Controller('credential-schema')
```

Comment:
REST compliant naming.

Reply:
This is WRT API spec given here - https://github.com/Sunbird-RC/sunbird-rc-core/blob/main/api-documentation/schema-api.yaml#L20